definitions/npm/react-helmet_v5.x.x/flow_v0.53.x-/react-helmet_v5.x.x.js

```diff
@@ -27,10 +27,15 @@ declare module 'react-helmet' {
     toComponent(): [React$Element<*>] | React$Element<*> | Array<Object>;
   }
 
+  declare interface AttributeTagMethods {
+    toString(): string;
+    toComponent(): {[string]: *};
```

Comment:
So toComponent function's return type is an object with one key? Can you link to the corresponding part of the react-helmet code where this return type is specified?

Reply:
Since `react-helmet` isn't typed at all, linking to where the return type is specified is a little hard. But as you can see [here](https://github.com/nfl/react-helmet/blob/master/src/HelmetUtils.js#L598), the cases of attributes for body and html are treated differently from the other methods by converting the tags to react props, i.e. not an array of react components.

The
```
{[string]: *}
```
is not an object with one key, but rather a map from string to `*` (see https://flow.org/en/docs/types/objects/#toc-objects-as-maps).

Reply:
Ah yes, that's right. Your solution looks good to me. Thanks for catching that!